src/altair/helpers.rs

```diff
@@ -69,9 +103,28 @@ pub fn get_next_sync_committee<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
-) -> Result<()> {
-    Ok(())
+    context: &Context,
+) -> Result<SyncCommittee<SYNC_COMMITTEE_SIZE>> {
+    // Return the next sync committee, with possible pubkey duplicates.
+    let indices = get_next_sync_committee_indices(state, context)?;
+    let public_keys = state
+        .validators
+        .iter()
+        .enumerate()
+        .filter_map(|(i, v)| {
+            if indices.contains(&i) {
+                Some(v.public_key.clone())
+            } else {
+                None
+            }
+        })
+        .collect::<Vector<_, SYNC_COMMITTEE_SIZE>>();
+    let aggregate_public_key = eth_aggregate_public_keys(&public_keys)?;
```

Comment:
if we change `eth_aggregate_public_keys` like I suggested below then I'm not sure we have `Deref` implemented for `Vector<T, _>` to `&[T]` but it would be useful imo